Source/WebCore/loader/ResourceMonitorChecker.cpp

```diff
@@ -42,37 +48,47 @@ ResourceMonitorChecker& ResourceMonitorChecker::singleton()
 ResourceMonitorChecker::ResourceMonitorChecker()
     : m_workQueue { WorkQueue::create("ResourceMonitorChecker Work Queue"_s) }
 {
+    protectedWorkQueue()->dispatchAfter(ruleListPreparationTimeout, [this] mutable {
+        if (m_ruleList)
+            return;
+
+        RESOURCEMONITOR_RELEASE_LOG("giving up to wait rule list");
```

Comment:
Maybe something like "did not receive rule list in time, using default eligibility" or something like that.